packages/IMAPClient-Core.package/ICFolder.class/instance/startUpdateProcess..st

```diff
@@ -1,10 +1,16 @@
 action - update emails
 startUpdateProcess: anUID
-
-	(self folderName = 'root') ifTrue: [^true].
 	
+	| movedEmails acc |
+	acc := self accountName.
```

Comment:
where do you use acc in this method? :sweat_smile:

Reply:
nowhere :D will be gone